packages/vscode-extension/src/extension.ts

```diff
@@ -746,7 +746,9 @@ function registerCodelensAndHoverProviders(context: vscode.ExtensionContext) {
   const manifestTemplateSelector = {
     language: "json",
     scheme: "file",
-    pattern: `**/${TemplateFolderName}/${AppPackageFolderName}/manifest.template.json`,
+    pattern: isV3Enabled()
```

Comment:
There might be problems if we only support manifest.template.json:
1. For SPFx, there's a manifest.template.local.json that also need codelens
2. For other templates that's customized by users, is it designed to not support?